extensions/wikia/AchievementsII/services/AchAwardingService.class.php

```diff
@@ -44,11 +44,16 @@ public function migration($user_id)  {
 	public function awardCustomNotInTrackBadge($user, $badge_type_id) {
 		wfProfileIn(__METHOD__);
 
-		global $wgExternalSharedDB;
+		global $wgExternalSharedDB, $wgAchievementsEditAddPhotoOnly;
+
+		if ( !empty( $wgAchievementsEditAddPhotoOnly ) && ( $badge_type_id != BADGE_EDIT && $badge_type_id != BADGE_PICTURE ) ) {
+			wfProfileOut(__METHOD__);
+			return;
+		}
 
 		$this->mUser = $user;
 
-		if( self::canEarnBadges( $this->mUser ) ) {
+		if ( self::canEarnBadges( $this->mUser ) ) {
```

Comment:
I try to format consistent at least inside a function. This file is already using inconsistent format everywhere. I can work on to fix it in this entire file.